### PR TITLE
Sticky-ad: Wait for 1s if ad is not stubbed

### PR DESCRIPTION
**Pull request by zhouyx**
Temp fix for #9123 
Related to #9126

### CODE REVIEW

File: extensions/amp-sticky-ad/1.0/amp-sticky-ad.js [hunk 1]
```diff
@@ -66,9 +70,28 @@ class AmpStickyAd extends AMP.BaseElement {
     this.ad_ = children[0];
     this.setAsOwner(this.ad_);
 
-    this.ad_.whenBuilt().then(() => {
-      this.mutateElement(() => {
-        toggle(this.element, true);
+    // TODO(@zhouyx, #9126): cleanup once research
+    // on custom-element stubbing is complete.
+    let customApiResolver;
+    const customApiPromise = new Promise(resolve => {
```
Comment (zhouyx) [inline extensions/amp-sticky-ad/1.0/amp-sticky-ad.js]: I changed it a bit because `timerFor().delay` does not return a promise.

Comment (jridgewell) [inline extensions/amp-sticky-ad/1.0/amp-sticky-ad.js]: `Timer#promise`

Comment (zhouyx) [inline extensions/amp-sticky-ad/1.0/amp-sticky-ad.js]: Ah! Only saw `timeoutPromise`. I'll switch to it

File: extensions/amp-sticky-ad/1.0/amp-sticky-ad.js [hunk 2]
```diff
@@ -66,9 +70,28 @@ class AmpStickyAd extends AMP.BaseElement {
...
+      customApiResolver = resolve;
+    });
+    if (this.ad_.whenBuilt) {
+      customApiResolver();
+    } else {
+      // Give 1s for amp-ad to stub. Report 1% error.
+      if (Math.random() < 0.01) {
+        dev().error(TAG, 'race condition on customElement stubbing');
+      }
+      timerFor(this.win).delay(() => {
```
Comment (dvoytenko) [inline extensions/amp-sticky-ad/1.0/amp-sticky-ad.js]: `timerFor(this.win).delay(customApiResolver, 1000)`

File: extensions/amp-sticky-ad/1.0/amp-sticky-ad.js (hunk 1 above)
Comment (zhouyx) [inline extensions/amp-sticky-ad/1.0/amp-sticky-ad.js]: Actually I'll clean this up anyway... decide not to switch.....

**Pull request merged**